.github/workflows/auto_branching.yml

```
@@ -104,6 +104,11 @@ jobs:
             rm -rf ./.github/workflows/dispatch_release.yml
             rm -rf ./.github/workflows/auto_branching.yml
 
+      - name: Remove CODEOWNERS
+        id: remove-codeowners
+        run: |
+            rm -rf ./.github/CODEOWNERS
```

Since CODEOWNERS is a file, not a directory, there is no need to delete it recursively.

```suggestion
            rm -f ./.github/CODEOWNERS
```

---

Sure, will update this, but there's no harm in keeping it